assets/templates/confirmation-page.html

```diff
@@ -58,7 +58,7 @@ <h1>Gracias por registrarte</h1>
         <span class="icon-registration m-bottom--lv6">Fíjate en tu correo electrónico</span>
         <p class="text-italic">* Al hacer clic en el botón que aparece en el Email, activarás tu cuenta y estarás listo para disfrutar todos los beneficios de Doppler.</p>
       </article>
-      <p>¿No has recibido el Email? <a href="#" class="link-green" target="_blank">Reenvíalo.</a></p>
+      <p>¿No has recibido el Email? <a href="#" target="_blank">Reenvíalo.</a></p>
```

Comment:
This changes should be reverted based on the comment about removing this class.

Reply:
I defined the default color the anchor in green, but I yes need a anchor with other color,  is necessary add class with color specific. With @santiagopaolucci we define this is correct and this modification no affect the others links in the webapp or aplicación. The anchor green is a standar the Doppler.